doc/tutorials/dnn/table_of_content_dnn.markdown

```diff
@@ -0,0 +1,35 @@
+The dnn Functionality (dnn module) {#tutorial_table_of_content_dnn}
+=====================================
+
+Here you will learn the about the basic building blocks of the library. A must read and know for
```

Comment:
Oops !
 I delete everything. Next commit

-   @subpage tutorial_dnn_googlenet

    *Compatibility:* \> OpenCV 3.2
    
    *Author:* 

    In this tutorial you will learn how to use opencv_dnn module for image classification by using GoogLeNet trained network from Caffe model zoo.

Do you know module author ?